### PR TITLE
Add SSL and Nginx feature

### PR DESCRIPTION
**Pull request by michaelmok2021**

### CODE REVIEW

File: backend-resources/src/main/resources/application.yml [hunk 1]
```diff
@@ -9,8 +12,8 @@ spring:
     oauth2:
       resourceserver:
         jwt:
-          issuer-uri: http://backend-auth:8070/realms/${realm}
-          jwk-set-uri: http://backend-auth:8070/realms/${realm}/protocol/openid-connect/certs
+          issuer-uri: https://sso.billview.com.au/realms/myfirsttest
```
Comment (HamishWa) [inline backend-resources/src/main/resources/application.yml:15]: @michaelmok2021 Is this the correct realm?

File: backend-resources/src/main/java/com/sergio/auth/backend/resources/controllers/ResourceController.java [hunk 2]
```diff
@@ -39,4 +36,53 @@ public Map<String, Object> getUserInfo(@AuthenticationPrincipal Jwt principal) {
...
+    public String mikeSays(@RequestBody String message) {
+
+        return mike(message);
+    }
+    @GetMapping(path= "/mike")
+    public String getMike(@AuthenticationPrincipal Jwt principal){
+//        log.info(principal.getClaimAsString("preferred_username") + " requests mike.");
+        return mike("hi.");
+    }
+    public String mike(String message){
```
Comment (HamishWa) [inline backend-resources/src/main/java/com/sergio/auth/backend/resources/controllers/ResourceController.java]: @michaelmok2021 I have modified this in the latest version.

**Review by HamishWa — CHANGES_REQUESTED** [review]
Needs to be updated to latest branch for the enviroment variables and resources.

File: backend-resources/src/main/resources/application.yml (hunk 1 above)
Comment (michaelmok2021) [reply backend-resources/src/main/resources/application.yml:15]: Yes

**Pull request merged**